net-misc/seafile-client/seafile-client-6.0.4-r1.ebuild

```diff
@@ -28,9 +28,11 @@ RDEPEND="net-libs/libsearpc
 	dev-qt/qtdbus:5
 	shibboleth? ( || ( dev-qt/qtwebengine:5[widgets] dev-qt/qtwebkit:5 ) )"
 DEPEND="${RDEPEND}
-	dev-qt/linguist-tools:5"
+	dev-qt/linguist-tools:5
+	test? ( dev-qt/qttest:5 )"
 
 src_prepare() {
+	use test || eapply "${FILESDIR}/${P}-only-use-qttest-when-needed.patch"
```

Comment:
You want to apply this unconditionally. After all, there's a conditional inside.